packages/vision/src/helpers.js

```diff
@@ -74,11 +74,12 @@ var coerceImage = (image, callback) => {
  *   asking for the single feature annotation.
  */
 var _createSingleFeatureMethod = featureValue => {
-  return function(image, options) {
-    return this.annotateImage({
+  return function(image, options = {}, callOptions = undefined) {
```

Comment:
The former is the way Node works, and there is nothing I can do about that.

I will let CI figure out the default parameters part. If it does not then I will change to the old `options = options || {}` idiom.